src/main/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpoint.java

```diff
@@ -77,8 +77,8 @@ private void validateMatchingCaseId(UUID caseId, UUID dtoCaseId, String dtoName)
       throws CTPException {
     if (!caseId.equals(dtoCaseId)) {
       String message =
-          "The caseid in the " + dtoName + " URL does not match the caseid in the request body";
-      log.with(caseId).warn(message);
+          "The caseid in the given dto URL does not match the caseid in the request body";
```

Comment:
"The path parameter caseId does not match the caseId in the request body"

Reply:
done

Reply:
"The path parameter caseId does not match the caseId in the request body" is clearer.